src/index.js

```diff
@@ -162,6 +162,10 @@ class HCaptcha extends React.Component {
 
         // render captcha
         this.renderCaptcha();
+
+        if (this.executeOnLoad) {
+          this.execute();
```

Comment:
You are going to want to reset the `this.executeOnLoad` after you call `this.execute`.